server/exposures/cmaq_utils.py

```diff
@@ -56,21 +53,53 @@ def latlon2rowcol(latitude, longitude, year):
         xorig = -2556000.0
         yorig = -1728000.0
 
-
         # Create the CMAQ projection so we can do some conversions
         proj_str = "+proj=" + proj + " +lon_0=" + str(lon_0) + " +lat_0=" + str(lat_0) + \
-               " +lat_1=" + str(lat_1) + " +lat_2=" + str(lat_2) + " +units=meters"
+                   " +lat_1=" + str(lat_1) + " +lat_2=" + str(lat_2) + " +units=meters"
 
         p1 = Proj(projparams=proj_str)
 
-        x1,y1 = p1(longitude, latitude)
+        x1, y1 = p1(longitude, latitude)
 
         # verify the points are in the grid
-        if((x1>=xorig) and (x1<=(xorig+(xcell*number_of_columns))) and
-           (y1>=yorig) and (y1<=(yorig+(ycell*number_of_rows)))):
-        
-            # find row and column in grid     
-            col_no=int(abs((xorig)-x1)/xcell) + 1
-            row_no=int((abs(yorig)+y1)/ycell) + 1
-
-        return row_no, col_no 
+        if ((x1 >= xorig) and (x1 <= (xorig + (xcell * number_of_columns))) and
+                (y1 >= yorig) and (y1 <= (yorig + (ycell * number_of_rows)))):
+            # find row and column in grid
+            col_no = int(abs((xorig) - x1) / xcell) + 1
+            row_no = int((abs(yorig) + y1) / ycell) + 1
+            
+    return row_no, col_no
+
+
+def cmaq_calc_score(exposure_type, value):
+    # calculate exposure score based on exposure_docs criteria
+    if exposure_type == 'pm25':
+        if value < 4.0:
+            score = 1
+        elif 4.0 <= value < 7.06:
+            score = 2
+        elif 7.06 <= value < 8.97:
+            score = 3
+        elif 8.97 <= value < 11.37:
+            score = 4
+        elif value >= 11.37:
```

Comment:
The condition ```value >= 11.37``` is not needed since it is implied by conditions above it. So just ```else``` is sufficient here.